base/download.jl

```diff
@@ -25,28 +25,16 @@ if Sys.iswindows()
     end
 else
     function download(url::AbstractString, filename::AbstractString)
-        global downloadcmd
-        if downloadcmd === nothing
-            for checkcmd in ("curl", "wget", "fetch")
-                try
-                    # Sys.which() will throw() if it can't find `checkcmd`
-                    Sys.which(checkcmd)
-                    downloadcmd = checkcmd
-                    break
-                catch
-                end
-            end
-        end
-        if downloadcmd == "wget"
+        if !isempty(Sys.which("wget"))
```

Comment:
shouldn't this be `Sys.which("wget") !== nothing`?

Reply:
Yes, not sure where I got the idea that empty strings were being returned